patterns/solid/src/main/java/com/baeldung/l/advanced/Car.java

```diff
@@ -0,0 +1,14 @@
+package com.baeldung.l.advanced;
+
+public abstract class Car {
+    // invariant: speed < limit;
+    protected int speed, limit;
```

Comment:
Should we have the `limit` declare in the more conventional method `protected int limit` rather than with this quirky form?

Reply:
Fixed